src/test/scala/io/qbeast/spark/index/model/transformer/CDFNumericQuantilesIndexingTest.scala

```diff
@@ -0,0 +1,116 @@
+package io.qbeast.spark.index.model.transformer
+
+import io.qbeast.core.transform.CDFNumericQuantilesTransformation
+import io.qbeast.spark.QbeastIntegrationTestSpec
+import io.qbeast.spark.QbeastTable
+
+class CDFNumericQuantilesIndexingTest
+    extends QbeastIntegrationTestSpec
+    with CDFQuantilesTestUtils {
+
+  "Quantiles index" should "allow append without specifying columnStats" in withSparkAndTmpDir(
+    (spark, tmpDir) => {
+
+      import spark.implicits._
+      val df = 1.to(10).toDF("int_col")
```

Comment:
Generate DataFrames with `val df = spark.range(1, 10).toDF(colName)` can is more efficient.